Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -683,6 +688,9 @@ private class PhotonActionSheetCell: UITableViewCell {
             stackView.addArrangedSubview(disclosureLabel)
         case .Disclosure:
             stackView.addArrangedSubview(disclosureIndicator)
+            disclosureIndicator.snp.makeConstraints { make in
+                make.width.equalTo(PhotonActionSheetUX.IconSize.width)
+            }
```

Comment:
I am seeing another bug with the disclosure icon shrinking in width. This fixes it.

Reply:
I didn't include a screenshot for this case. I'll try compression resistance .required, as long as the intrinsic size is correctly set to the icon width automatically when the icon is loaded it should work.